lib/registrant.rb

```diff
@@ -0,0 +1,25 @@
+class Registrant
+    attr_reader :name, :age, :permit, :license_data
+    def initialize(name, age, permit = nil)
+        @name = name
+        @age = age
+        @permit = permit unless permit.nil? do
+        end
+        @license_data = {:written => false, :license => false, :renewed => false}
+    end
+
+    def permit?
+        if permit == nil
+        return false
+        else 
+            permit == true
+        return true
+        end
```

Comment:
This does work logic-wise. 
However, if we already have @permit = false in the initialize method, then the inside of this method could just be `@permit` so it acts like an attr_reader.